### DISCUSSION TITLE
用了在线使用说明添加字体之后，章节标题加粗消失了，并且字体发生了改变

### DISCUSSION THREAD

**Discussion by jpDeng111** [sjtug/SJTUThesis#659] [提问] [answered]
https://github.com/sjtug/SJTUThesis/wiki/%E5%9C%A8%E7%BA%BF%E4%BD%BF%E7%94%A8%E8%AF%B4%E6%98%8E
按照上面的操作之后，章节标题加粗消失了，并且正文的字体和原来不太一样，不知道是哪里出了问题？（但是不这么做的话，我的名字就打不出来）

代码的最后四行因为报错字体已经存在，就把它们注释掉了

**Comment by jpDeng111** [comment]
操作系统是windows，使用的是overleaf的版本，模板是新的SJTU1.0.0

**Reply by jpDeng111** [reply]
然后编译日志我不知道从哪里找，我截个图好了

**Comment by skyzh** [comment]
提供的在线字体是不自带粗体字重的。您需要使用最新的 develop 分支才可以正常加粗。

**Reply by skyzh** [reply]
刚刚直接把 develop 分支上传到了 latex.sjtu，可以直接使用，不用做其他设置。图中两个字都是之前同学反馈的生僻字。

<img width="559" alt="image" src="https://user-images.githubusercontent.com/4198311/119773007-a2f10680-bef2-11eb-8bb4-1c6a40c65911.png">

**Reply by jpDeng111** [reply]
啊谢谢！我试试

**Reply by jpDeng111** [reply]
你好Alex！很不好意思又来打扰你，我注册了之后把项目重新上传到了latex.sjtu.edu.cn上面，但是旸旻这两个字我输入之后什么都没有，是我漏了什么操作的缘故吗？

**Reply by skyzh** [reply]
下载 https://github.com/sjtug/SJTUThesis/archive/refs/heads/develop.zip 试试看？

**Reply by jpDeng111** [reply]
啊谢谢！非常感谢！

**Comment by AlexaraWu** [comment, marked as answer]
测试了一下发现学校的 LaTeX 助手使用的镜像安装了 Windows 的中文字体。这样就简单了，不需要额外的上传配置字体。

使用正式版模板的话，添加模板选项 `fontset=windows` 就可以了； `develop` 分支开发版重写了 `ctex` 的中文字体配置，可以直接用。

**Reply by AlexaraWu** [reply]
另外目前开发版在 Overleaf 上会默认使用思源宋体而非 Fandol 字体，一般情况下不会有缺字的问题。
